tasks/main.yml

```diff
@@ -117,40 +99,18 @@
   when:
     - deploydrupal_site_install
 
-# Import the latest configuration. This includes the latest
-# configuration_split configuration. Importing this twice ensures that the
-# latter command enables and disables modules based upon the most up to date
-# configuration.
-- name: import configuration 1/2.
-  shell: "{{ deploydrupal_drush_path }} config-import -y"
-  args:
-    chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
-  register: drush_output
-  when:
-    - deploydrupal_config_import
-
-- name: print drush output
```

Comment:
Do we need this? It seems that this just duplicates output in many situations.

Reply:
This is printing the output for the above drush command, which we removed.